lm_eval/tasks/calamita/MT_CALAMITA/utils.py

```diff
@@ -21,25 +21,55 @@ def _search_delimiters(model_output: str) -> str:
     if right_delimiter in model_output:
         end = model_output.find(right_delimiter)
 
+    if len(model_output) < 1:
+        return "---"  # empty string as a replacement
     return model_output[start:end].replace('<', '').replace('>', '').strip()
 
+def _check_error_input(ref, pred):
```

Comment:
Scoring functions from the evaluate library do not support empty inputs. For this reason, this function protects the evaluation procedure in case `len(input) < 1`. If the model doesn't generate anything the assigned score will be 0